test/WebJobs.Script.Tests/Binding/BlobBindingTests.cs

```diff
@@ -19,54 +19,54 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Binding
 {
     public class BlobBindingTests
     {
-        [Theory]
-        [InlineData(typeof(ICloudBlob), FileAccess.Read, FileAccess.ReadWrite)]
-        [InlineData(typeof(CloudBlockBlob), FileAccess.Read, FileAccess.ReadWrite)]
-        [InlineData(typeof(CloudPageBlob), FileAccess.Read, FileAccess.ReadWrite)]
-        [InlineData(typeof(CloudBlobDirectory), FileAccess.Read, FileAccess.ReadWrite)]
-        [InlineData(typeof(ICloudBlob), FileAccess.Write, FileAccess.ReadWrite)]
-        [InlineData(typeof(CloudBlockBlob), FileAccess.Write, FileAccess.ReadWrite)]
-        [InlineData(typeof(CloudPageBlob), FileAccess.Write, FileAccess.ReadWrite)]
-        [InlineData(typeof(CloudBlobDirectory), FileAccess.Write, FileAccess.ReadWrite)]
-        [InlineData(typeof(Stream), FileAccess.Write, FileAccess.Write)]
-        [InlineData(typeof(Stream), FileAccess.Read, FileAccess.Read)]
-        [InlineData(typeof(TextReader), FileAccess.Read, FileAccess.Read)]
-        [InlineData(typeof(TextWriter), FileAccess.Write, FileAccess.Write)]
-        [InlineData(typeof(string), FileAccess.Read, FileAccess.Read)]
-        public void GetCustomAttributes_WithReadWriteTypes_AppliesReadWriteAccess(Type parameterType, FileAccess metadataAccess, FileAccess expectedAccess)
-        {
-            var metadata = new BlobBindingMetadata()
-            {
-                Path = "my/blob"
-            };
-            var binding = new BlobBinding(new ScriptHostConfiguration(), metadata, metadataAccess);
+        //[Theory]
+        //[InlineData(typeof(ICloudBlob), FileAccess.Read, FileAccess.ReadWrite)]
```

Comment:
Yes, I've deleted the test. The blob binding provider no longer does any type specific conversions. ScriptBindingContext maps "direction" to a FileAccess, so if the user sets that direction properly, the correct access is used for the BlobAttribute. I've added ScriptBindingContextTests, and also updated the BlobTrigger sample for this scenario - once I changed the parameters to CloudBlockBlob, it failed to compile until I changed the direction to "inout".